src/Command/PlatformDelete.php

```diff
@@ -2,25 +2,21 @@
 
 namespace EclipseGc\CommonConsole\Command;
 
-use EclipseGc\CommonConsole\Platform\PlatformCommandTrait;
+use EclipseGc\CommonConsole\Exception\MissingPlatformException;
 use EclipseGc\CommonConsole\Platform\PlatformStorage;
-use EclipseGc\CommonConsole\PlatformCommandInterface;
 use EclipseGc\CommonConsole\PlatformInterface;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Question\ConfirmationQuestion;
-use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 
 /**
- * Class PlatformDelete
+ * Class PlatformDelete.
  *
  * @package EclipseGc\CommonConsole\Command
  */
-class PlatformDelete extends Command implements PlatformCommandInterface {
-
-  use PlatformCommandTrait;
```

Comment:
I felt like is in an unnecessary overhead. Since these are just simple tasks and we have PlatformStorage at our disposal, these commands can be more efficient and lightweight, nonetheless they don't depend on event dispatcher. Also I wanted to unify the approach across all commands hence the modification of the PlatformDelete command. Please let me know what you think about this simplification.

Reply:
Ok, I'm not sure I see how removing the trait/interface from the command make it any simpler. Maybe we could move the getExpectedPlatformOptions() implementation here into the trait? That way we have a sane default we can override as needed? It's just that the alias loading exists for exactly this sort of situation, so it seems odd not to use it.

Reply:
My motivation here was that if not necessary I wouldn't introduce another dependency unless it has an effective purpose. If we require the alias parameter anyway, the storage is capable of loading it if the configuration exists so we can omit the unecessary code execution carried out through multiple event dispatching.